zipline/utils/cache.py

```diff
@@ -350,7 +373,7 @@ def getpath(self, *path_parts):
     def _commit(self):
         """Sync the temporary directory to the final path.
         """
-        copytree(self.path, self._final_path)
+        dir_util.copy_tree(self.path, self._final_path, verbose=1)
```

Comment:
copytree() fails if the target directory already exists on py2.7 but not 3.4, dir_util.copy_tree is okay with merging on both versions of python